deploy/helm/metacontroller/templates/pdb.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if and .Values.podDisruptionBudget.enabled (gt (.Values.replicas | int) 1) }}
```

Comment:
with the change to values.yaml, this could become
```
{{- if .Values.podDisruptionBudget }}
```
up to you if you also want to include the replica check, I don't have a strong opinion

Reply:
I prefer to have the replica check. 
The statefullset, which is used by metacontroller, won't update if it isn't allowed to bring at least one pod down.

A deployment can do a surge but a statefullset won't.